contracts/contribute/ContributeManager.sol

```diff
@@ -300,7 +359,7 @@ contract ContributeManager is ERC721TokenReceiver {
     }
 
     /// @dev Unstakes service Id corresponding to the msg.sender and clears the contributor record.
-    function unstake() external {
+    function unstake(bool pullService) external {
```

Comment:
Choose whether the service is kept by CM or transferred to `msg.sender` after being unstaked, terminated and unbonded.

Reply:
why are we adding this? It just makes everything more complex and opens more opportunities for error.

Reply:
If we need to have unstake in two steps for whatever reason then I suggest EVERYONE needs to use pullService as a second step..

Reply:
We don't need to unstake in 2 steps. When unstaked - the service is terminated and unbonded. To pull it to the user is just an option, which we can scratch, and keep those services forever by CM.

Reply:
They are fake services anyway, and msg.sender has the access to the multisig by default.

Reply:
For cleanliness I'm ok with keeping the pull logic.